crates/tauri/src/manager/mod.rs

```diff
@@ -221,6 +221,13 @@ pub struct AppManager<R: Runtime> {
   pub(crate) invoke_key: String,
 
   pub(crate) channel_interceptor: Option<ChannelInterceptor<R>>,
+
+  // the value is set to true when the event loop is already exited
+  event_loop_exit_mutex: Mutex<bool>,
+  event_loop_exit_condvar: std::sync::Condvar,
```

Comment:
i'm not super familiar with condvar, is there an advantage in using this over e.g. a multi producer multi consumer channel?

Reply:
AFAIK, mcmc channel can only one thread at once but condvar can wake multiple threads at once so I used condvar